packages/cli/src/commands/generate/page/__tests__/page.test.js

```diff
@@ -377,13 +380,15 @@ test('file generation', async () => {
 
   global.mockFs = true
 
+  silenceConsoleOutput()
   await page.handler({
     name: 'HomePage', // 'Page' should be trimmed from name
     path: '',
     force: false,
     tests: true,
     stories: true,
   })
+  restoreConsoleOutput()
```

Comment:
I wish it was this easy, but unfortunately it's not. If `handler` throws an error the console is never restored. Only reliable solution I've come up with is to wrap the test in a nested `describe` block, and have `beforeEach` inside that block.

Reply:
Ah good point of course I can't rely on it not throwing. Are you against that suggestion of using the nested describe?

Reply:
No, not at all. Please go ahead with that 🙂

Reply:
Hopefully my latest changes fix this.

Reply:
No worries, I'll have a look